pkg/resource/podtemplatespec.go

```diff
@@ -254,6 +254,22 @@ func makePodTemplateSpec(coreClient coreset.CoreV1Interface, driver storage.Driv
 			Volumes:            volumes,
 			ServiceAccountName: params.Pod.ServiceAccount,
 			SecurityContext:    securityContext,
+			Affinity: &corev1.Affinity{
+				PodAntiAffinity: &corev1.PodAntiAffinity{
+					PreferredDuringSchedulingIgnoredDuringExecution: []corev1.WeightedPodAffinityTerm{
+						{
+							Weight: 100,
+							PodAffinityTerm: corev1.PodAffinityTerm{
+								//TODO godoc for this field says it cannot be empty, but the doc at
+								// https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#an-example-of-a-pod-that-uses-pod-affinity
+								// talks about using an empty topologyKey with anti-affinity as signifying "all topologies",
+								// That said, the standard kubernetes.io/hostname has appeared sufficient in testing on AWS clusters with 3 worker nodes
+								TopologyKey: "kubernetes.io/hostname",
```

Comment:
You can use `params.Deployment.Namespace` to get the namespace.

Reply:
update pushed @dmage ... thanks